src/test/scala/zio/interop/TwitterSpec.scala

```diff
@@ -50,6 +50,41 @@ object TwitterSpec extends DefaultRunnableSpec {
           task.map(b => assert(b)(isTrue) && assert(value.get())(equalTo(0)))
         }
       ),
+      suite("Deprecated Task.fromTwitterFuture")(
```

Comment:
I would remove the entire suite, as the comment above would make it effectively useless.